updating/index.adoc

```diff
@@ -0,0 +1,64 @@
+:_content-type: ASSEMBLY
+[id="updating-clusters-overview"]
+= Updating clusters overview
+:context: updating-clusters-overview
+
+toc::[]
+
+You can update a {product-title} cluster with a single operation by using the web console or the OpenShift CLI (oc) with {product-title} 4. For more information, see xref:../updating/understanding-openshift-updates.adoc#understanding-openshift-updates[Understanding {product-title} updates].
+
+[id='updating-clusters-overview-understand-update-service']
+== Understanding the OpenShift Update Service
+xref:../updating/updating-restricted-network-cluster.adoc#update-restricted-network-cluster-update-service[Understanding the {product-title} Update Service]: For clusters with internet accessibility, Red Hat provides over-the-air updates through an {product-title} update service as a hosted service located behind public APIs. For more information, see the following information:
+
+* xref:../updating/updating-restricted-network-cluster.adoc#update-service-overview_updating-restricted-network-cluster[About the OpenShift Update Service]
```

Comment:
[About the OpenShift Update Service] is linked to https://github.com/openshift/openshift-docs/blob/c7bcafb96766f2c5bcde1e31c4019b2b315b3e4a/updating/updating-restricted-network-cluster.adoc#update-service-overview_updating-restricted-network-cluster wrongly. I don't think it's only related with restricted cluster update. Could you review the mapping with 4.6 doc?

Reply:
Addressed